sparsify/__main__.py

```diff
@@ -144,42 +143,36 @@ def run():
 
     if ddp:
         torch.cuda.set_device(int(local_rank))
-
-        # Increase the default timeout in order to account for slow downloads
-        # and data preprocessing on the main rank
```

Comment:
Generally we want to avoid touching unrelated comments from other PRs unless they're clearly unneeded/inaccurate comments. All the comments in this block seem valuable to me